src/drm.cpp

```diff
@@ -443,6 +443,7 @@ struct drm_t {
 
 	struct liftoff_device *lo_device;
 	struct liftoff_output *lo_output;
+	struct liftoff_output_apply_options *lo_options;
```

Comment:
I don't think we need to store a pointer to the options? Can't we just declare the options right before calling `liftoff_output_apply()`?

Reply:
Yeah

Reply:
Done